staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -693,6 +724,29 @@ type PersistentVolumeClaimStatus struct {
 	// +mapType=granular
 	// +optional
 	AllocatedResourceStatuses map[ResourceName]ClaimResourceStatus `json:"allocatedResourceStatuses,omitempty" protobuf:"bytes,7,rep,name=allocatedResourceStatuses"`
+	// volumeAttributesClassName is the name of VolumeAttributesClass which is latest applied by
+	// the CSI driver. This field is not set by default but when modification is complete,
+	// it's set to the applied VolumeAttributesClass.
+	// This is an alpha field and requires enabling VolumeAttributesClass feature.
+	// +featureGate=VolumeAttributesClass
+	// +optional
+	VolumeAttributesClassName *string `json:"volumeAttributesClassName,omitempty" protobuf:"bytes,8,opt,name=volumeAttributesClassName"`
+	// VolumeAttributesModifyStatus stores status of modification operation. It can be in any of following states:
+	//  - PersistentVolumeClaimControllerModifyVolumeInProgress
+	//    State set when modify volume controller starts modifying the volume in control-plane.
```

Comment:
I'm having trouble understanding this.  Is this saying that the condition is set to this state when the controller observes that the volume attribute class has change and is in progress of modifying it? (I'm hoping that we capture the clarification here in the documentation)

Reply:
The condition is set to this state when the controller observes that the volume attribute class has change and is in progress of modifying it or waiting for driver installed and vac created.